content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/least_packages.rst

```diff
@@ -6,7 +6,7 @@ The *Least Packages* removal strategy fulfills an order by opening the fewest nu
 which is ideal for maintaining organized stock without needing to open multiple boxes.
 
 .. seealso::
-   - :doc:`Other removal strategies <removal>`
+   - :doc:`About removal strategies <../removal_strategies>`
```

Comment:
Technically, changing the wording would be part of another commit since it could also be applied to older versions than 17.0. But this is such a minor change that you can disregard my comment. It's mostly to keep in mind that when we apply different types of changes, they may apply to different versions.